### DISCUSSION TITLE
Recognizing All Contributors

### DISCUSSION THREAD

**Discussion by jhuleatt** [FirebaseExtended/reactfire#241] [General]
The ReactFire repo now uses the [All Contributors spec](https://allcontributors.org/docs/en/overview) to recognize the contributions of everyone that has helped out with ReactFire v2.

**Comment by jhuleatt** [comment]
To add yourself to the contributors list, invoke the bot:

@all-contributors, please add @jhuleatt for code, doc, example, ideas, tutorial, talk

**Comment by jhuleatt** [comment]
Testing the bot:

@all-contributors, please add @jhuleatt for code

**Comment by jhuleatt** [comment]
@all-contributors please add @jhuleatt for code

**Comment by allcontributors** [comment]
@jhuleatt 

I've put up [a pull request](https://github.com/FirebaseExtended/reactfire/pull/242) to add @jhuleatt! :tada:

**Comment by jhuleatt** [comment]
Working through some permissions issues with the bot's PR https://github.com/FirebaseExtended/reactfire/pull/242

**Comment by w3bdesign** [comment]
@all-contributors please add @w3bdesign for code

**Comment by allcontributors** [comment]
@w3bdesign 

I've put up [a pull request](https://github.com/FirebaseExtended/reactfire/pull/246) to add @w3bdesign! :tada:

**Comment by prescottprue** [comment]
@all-contributors please add @prescottprue for code

**Comment by allcontributors** [comment]
@prescottprue 

I've put up [a pull request](https://github.com/FirebaseExtended/reactfire/pull/247) to add @prescottprue! :tada:
